src/gui/fspathedit_p.h

```diff
@@ -132,11 +132,9 @@ namespace Private
         void keyPressEvent(QKeyEvent *event) override;
         void contextMenuEvent(QContextMenuEvent *event) override;
 
-    private slots:
-        void showCompletionPopup();
-
     private:
         static QString warningText(FileSystemPathValidator::TestResult r);
+        void showCompletionPopup();
```

Comment:
No. But anyway the new  connection style does not require a method to be pre-processed by the moc compiler.